scripts/submit_csv.sh

```diff
@@ -46,6 +46,8 @@ sed "
   s|%COPYRECO%|${COPYRECO:-}|g;
   s|%COPYFULL%|${COPYFULL:-}|g;
   s|%COPYLOG%|${COPYLOG:-}|g;
+  s|%USERUCIO%|${USERUCIO:-}|g;
+  s|%X509_USER_PROXY%|${$(basename $X509_USER_PROXY):-}|g;
```

Comment:
Why go with basename? It means the same variable name means different things depending on context.

Reply:
I don't think i thought this through properly. I was trying to deal with the scenario where someone specifies the local certificate directory where it's not the current working directory. 
X509_USER_PROXY=location/x509_user_proxy when transferred by condor will sit at x509_user_proxy in the remote node. Wouldn't i need to strip off the local directory information.

Reply:
Will we ever want an absolute path for X509_USER_PROXY? `/cvmfs/eic.opensciencegrid.org/certs/token`? Anyway, feel free to resolve, we can figure out if this is limiting us when it becomes relevant.